<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Recommendations and solutions for generalizable single-cell perturbation response prediction obtained from a systematic benchmark</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/yeti.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />
<script>(function(f,b){if(!b.__SV){var e,g,i,h;window.mixpanel=b;b._i=[];b.init=function(e,f,c){function g(a,d){var b=d.split(".");2==b.length&&(a=a[b[0]],d=b[1]);a[d]=function(){a.push([d].concat(Array.prototype.slice.call(arguments,0)))}}var a=b;"undefined"!==typeof c?a=b[c]=[]:c="mixpanel";a.people=a.people||[];a.toString=function(a){var d="mixpanel";"mixpanel"!==c&&(d+="."+c);a||(d+=" (stub)");return d};a.people.toString=function(){return a.toString(1)+".people (stub)"};i="disable time_event track track_pageview track_links track_forms track_with_groups add_group set_group remove_group register register_once alias unregister identify name_tag set_config reset opt_in_tracking opt_out_tracking has_opted_in_tracking has_opted_out_tracking clear_opt_in_out_tracking start_batch_senders people.set people.set_once people.unset people.increment people.append people.union people.track_charge people.clear_charges people.delete_user people.remove".split(" ");
for(h=0;h<i.length;h++)g(a,i[h]);var j="set set_once union unset remove delete".split(" ");a.get_group=function(){function b(c){d[c]=function(){call2_args=arguments;call2=[c].concat(Array.prototype.slice.call(call2_args,0));a.push([e,call2])}}for(var d={},e=["get_group"].concat(Array.prototype.slice.call(arguments,0)),c=0;c<j.length;c++)b(j[c]);return d};b._i.push([e,f,c])};b.__SV=1.2;e=f.createElement("script");e.type="text/javascript";e.async=!0;e.src="undefined"!==typeof MIXPANEL_CUSTOM_LIB_URL?
MIXPANEL_CUSTOM_LIB_URL:"file:"===f.location.protocol&&"//cdn.mxpnl.com/libs/mixpanel-2-latest.min.js".match(/^\/\//)?"https://cdn.mxpnl.com/libs/mixpanel-2-latest.min.js":"//cdn.mxpnl.com/libs/mixpanel-2-latest.min.js";g=f.getElementsByTagName("script")[0];g.parentNode.insertBefore(e,g)}})(document,window.mixpanel||[]);
mixpanel.init("45da4dcd73c9f511fc152d093652d0ec", {"api_host": "https://api-eu.mixpanel.com", batch_requests: true})</script>
<script>mixpanel.track("Open page")</script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">



<div id="nav-placeholder"></div>

<script>
$(function(){
  $("#nav-placeholder").load("navbar-content.html");
});
</script>

<div id="header">



<h1 class="title toc-ignore">Recommendations and solutions for
generalizable single-cell perturbation response prediction obtained from
a systematic benchmark</h1>

</div>


<p>Various computational methods have been developed to predict
perturbation effects, but despite claims of promising performance,
concerns about the true efficacy of these models continue to be raised,
particularly when the models are evaluated across diverse unseen
cellular contexts and unseen perturbations. To address this, a
comprehensive benchmark was conducted for 21 single-cell perturbation
response prediction methods, including methods concerning genetic and
chemical perturbations; 29 datasets were used, and various evaluation
metrics were applied to assess the generalizability of the methods to
unseen cellular contexts and perturbations. Recommendations regarding
the method limitations, method generalization and method selection were
obtained. Finally, an applicable solution that leverages prior knowledge
through cellular context embedding to improve the generalizability of
models to new cellular contexts is presented.</p>
<p><a data-fancybox href="figures/index/fig1.png">
<img src="figures/index/fig1.png"> </a></p>
<p><br/></p>
<div id="benchmark-methods" class="section level2">
<h2>Benchmark methods</h2>
<p>All benchmark methods analyzed in our study are listed below. Details
of the setting were available in our manuscript.</p>
<div id="ygjhmdkdmb" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#ygjhmdkdmb table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#ygjhmdkdmb thead, #ygjhmdkdmb tbody, #ygjhmdkdmb tfoot, #ygjhmdkdmb tr, #ygjhmdkdmb td, #ygjhmdkdmb th {
  border-style: none;
}

#ygjhmdkdmb p {
  margin: 0;
  padding: 0;
}

#ygjhmdkdmb .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#ygjhmdkdmb .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#ygjhmdkdmb .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#ygjhmdkdmb .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#ygjhmdkdmb .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#ygjhmdkdmb .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ygjhmdkdmb .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#ygjhmdkdmb .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#ygjhmdkdmb .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#ygjhmdkdmb .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#ygjhmdkdmb .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#ygjhmdkdmb .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#ygjhmdkdmb .gt_spanner_row {
  border-bottom-style: hidden;
}

#ygjhmdkdmb .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#ygjhmdkdmb .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#ygjhmdkdmb .gt_from_md > :first-child {
  margin-top: 0;
}

#ygjhmdkdmb .gt_from_md > :last-child {
  margin-bottom: 0;
}

#ygjhmdkdmb .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#ygjhmdkdmb .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#ygjhmdkdmb .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#ygjhmdkdmb .gt_row_group_first td {
  border-top-width: 2px;
}

#ygjhmdkdmb .gt_row_group_first th {
  border-top-width: 2px;
}

#ygjhmdkdmb .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#ygjhmdkdmb .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#ygjhmdkdmb .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#ygjhmdkdmb .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ygjhmdkdmb .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#ygjhmdkdmb .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#ygjhmdkdmb .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#ygjhmdkdmb .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#ygjhmdkdmb .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ygjhmdkdmb .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#ygjhmdkdmb .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#ygjhmdkdmb .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#ygjhmdkdmb .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#ygjhmdkdmb .gt_left {
  text-align: left;
}

#ygjhmdkdmb .gt_center {
  text-align: center;
}

#ygjhmdkdmb .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#ygjhmdkdmb .gt_font_normal {
  font-weight: normal;
}

#ygjhmdkdmb .gt_font_bold {
  font-weight: bold;
}

#ygjhmdkdmb .gt_font_italic {
  font-style: italic;
}

#ygjhmdkdmb .gt_super {
  font-size: 65%;
}

#ygjhmdkdmb .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#ygjhmdkdmb .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#ygjhmdkdmb .gt_indent_1 {
  text-indent: 5px;
}

#ygjhmdkdmb .gt_indent_2 {
  text-indent: 10px;
}

#ygjhmdkdmb .gt_indent_3 {
  text-indent: 15px;
}

#ygjhmdkdmb .gt_indent_4 {
  text-indent: 20px;
}

#ygjhmdkdmb .gt_indent_5 {
  text-indent: 25px;
}

#ygjhmdkdmb .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#ygjhmdkdmb div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>
<table class="gt_table" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
  <thead>
    <tr class="gt_col_headings">
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="a-Method"> Method</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="Generalization">Generalization</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="Article">Article</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="Time">Time</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="Title">Title</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="Version">Version</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/nitzanlab/biolord">biolord</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Cellular context/Perturbation</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://www.nature.com/articles/s41587-023-02079-x">Nature Biotechnology</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2024</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>Disentanglement of single-cell data with biolord</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.3</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/bunnech/cellot">CellOT</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Cellular context</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://www.nature.com/articles/s41592-023-01969-x">Nature Methods</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2023</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>Learning single-cell perturbation responses using neural optimal transport</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.1</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/LiouCharlie/INVAE/">inVAE</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Cellular context</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://www.mdpi.com/2306-5354/10/9/996">Bioengineering</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2023</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>Homogeneous Space Construction and Projection for Single-Cell Expression Prediction Based on Deep Learning</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.1</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/ZhangLabGT/scDisInFact">scDisInFact</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Cellular context</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://www.nature.com/articles/s41467-024-45227-w">Nature Communications</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2024</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>scDisInFact: disentangled learning for integration and prediction of multi-batch multi-condition single-cell RNA-sequencing data</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.1.0</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/theislab/scgen">scGen</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Cellular context</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://www.nature.com/articles/s41592-019-0494-8">Nature Methods</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2019</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>scGen predicts single-cell perturbation responses</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>2.1.0</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/jiang-q19/scPRAM">scPRAM</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Cellular context</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://academic.oup.com/bioinformatics/article/40/5/btae265/7646141">Bioinformatics</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2024</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>scPRAM accurately predicts single-cell gene expression perturbation response based on attention mechanism</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.1</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/XiajieWei/scPreGAN">scPreGAN</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Cellular context</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://academic.oup.com/bioinformatics/article/38/13/3377/6593485">Bioinformatics</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2022</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>scPreGAN, a deep generative model for predicting the response of single-cell expression to perturbation</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.1</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/Califorya/SCREEN">SCREEN</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Cellular context</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://link.springer.com/article/10.1007/s11704-024-31014-9">Frontiers of Computer Science</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2024</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>SCREEN: predicting single-cell gene expression perturbation responses via optimal transport</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.1</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/BhattacharyaLab/scVIDR">scVIDR</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Cellular context</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://www.cell.com/patterns/fulltext/S2666-3899(23)00186-1">Patterns</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2023</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>Generative modeling of single-cell gene expression for dose-dependent chemical perturbations</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.3</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/theislab/trVAE">trVAE</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Cellular context</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://academic.oup.com/bioinformatics/article/36/Supplement_2/i610/6055927">Bioinformatics</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2020</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>Conditional out-of-distribution generation for unpaired data using transfer VAE</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>1.1.2</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/BaiDing1234/AttentionPert">AttentionPert</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Perturbation</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://academic.oup.com/bioinformatics/article/40/Supplement_1/i453/7700899">Bioinformatics</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2021</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>AttentionPert: Accurately Modeling Multiplexed Genetic Perturbations with Multi-scale Effects</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.1</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/theislab/cpa">CPA</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Perturbation</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://www.embopress.org/doi/full/10.15252/msb.202211517">Molecular Systems Biology</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2023</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>Predicting cellular responses to complex perturbations in high-throughput screens</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.8.5</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/snap-stanford/GEARS">GEARS</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Perturbation</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://www.nature.com/articles/s41587-023-01905-6">Nature Biotechnology</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2022</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>Predicting transcriptional outcomes of novel multigene perturbations with GEARS</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.1.0</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/zou-group/GenePert">GenePert</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Perturbation</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://www.biorxiv.org/content/10.1101/2024.10.27.620513v1">bioRxiv</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2024</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>GenePert: Leveraging GenePT Embeddings for Gene Perturbation Prediction</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.1</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/const-ae/linear_perturbation_prediction-Paper">linearModel</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Perturbation</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://doi.org/10.1101/2024.09.16.613342">bioRxiv</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2024</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>Deep learning-based predictions of gene perturbation effects do not yet outperform simple linear methods</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.1</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/bowang-lab/scGPT">scGPT</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Perturbation</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://www.nature.com/articles/s41592-024-02201-0">Nature Methods</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2024</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>scGPT: toward building a foundation model for single-cell multi-omics using generative AI</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.2.1</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/biomap-research/scFoundation">scFoundation</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Perturbation</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://www.nature.com/articles/s41592-024-02305-7">Nature Methods</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2024</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>Large-scale foundation model on single-cell transcriptomics</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.1</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/theislab/chemCPA">chemCPA</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Perturbation</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://arxiv.org/abs/2204.13545">arXiv</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2022</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>Predicting Cellular Responses to Novel Drug Perturbations at a Single-Cell Resolution</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>2.0.0</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/PancakeZoy/scouter">scouter</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Perturbation</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://doi.org/10.1101/2024.12.06.627290">bioRxiv</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2024</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>Scouter: Predicting Transcriptional Responses to Genetic Perturbations with LLM embeddings</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.1</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/HelloWorldLTY/scELMo">scELMo</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Perturbation</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://arxiv.org/abs/2004.13214">bioRxiv</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2024</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>scELMo: Embeddings from Language Models are Good Learners for Single-cell Data Analysis</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.1</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/xCompass-AI/geneCompass">GeneCompass</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Perturbation</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://www.nature.com/articles/s41422-024-01034-y">Cell Research</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2024</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>GeneCompass: deciphering universal gene regulatory mechanisms with a knowledge-informed cross-species foundation model</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.1</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/hliulab/cycleCDR">cycleCDR</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Perturbation</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://doi.org/10.1093/bioinformatics/btae248">bioRxiv</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2024</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>Predicting single-cell cellular responses to perturbations using cycle consistency learning</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.1</span></td></tr>
    <tr><td headers=" Method" class="gt_row gt_left"><span class='gt_from_md'><a href="https://github.com/Perturbation-Response-Prediction/PRnet">PRnet</a></span></td>
<td headers="Generalization" class="gt_row gt_left"><span class='gt_from_md'>Perturbation</span></td>
<td headers="Article" class="gt_row gt_left"><span class='gt_from_md'><a href="https://www.nature.com/articles/s41467-024-53457-1">Nature communication</a></span></td>
<td headers="Time" class="gt_row gt_left"><span class='gt_from_md'>2024</span></td>
<td headers="Title" class="gt_row gt_left"><span class='gt_from_md'>Predicting transcriptional responses to novel chemical perturbations using deep generative model for drug discovery</span></td>
<td headers="Version" class="gt_row gt_left"><span class='gt_from_md'>0.0.1</span></td></tr>
  </tbody>
  
  
</table>
</div>
</div>
<div id="benchmark-datasets-summary" class="section level2">
<h2>Benchmark datasets summary</h2>
<p>All datasets analyzed in our study are listed in the <a
href="figures/index/fig1.png">Workflow</a>. We have uploaded all
benchmark datasets to Figshare and Zenodo, which can be obtained from <a
href="https://doi.org/10.6084/m9.figshare.28143422">Figshare-Cellular</a>,
<a
href="https://doi.org/10.6084/m9.figshare.28147883">Figshare-Perturbation</a>,
<a href="https://doi.org/10.5281/zenodo.14607156">Zenodo-Cellular</a>
and <a
href="https://10.5281/zenodo.14638779">Zenodo-perturbation</a>.</p>
<hr style="border: none; height: 1px; background-color: #333333; margin-top: 1em; margin-bottom: 1em;">
<div
style="display: flex; justify-content: space-around; align-items: center; flex-wrap: wrap; padding: 10px;">
<p><a href="[https://github.com/bm2-lab/scPerturBench/](https://github.com/bm2-lab/scPerturBench/)" target="_blank" title="scPerturb" style="margin: 10px;">
<img src="figures/index/scPerturb.png" alt="scPerturb Logo" style="width:150px; height:auto; border:0;">
</a></p>
<p><a href="[http://www.perturbase.cn/](http://www.perturbase.cn/)" target="_blank" title="PerturBase" style="margin: 10px;">
<img src="figures/index/perturbase.png" alt="PerturBase Logo" style="width:150px; height:auto; border:0;">
</a></p>
<p><a href="[https://github.com/theislab/sc-pert/](https://github.com/theislab/sc-pert/)" target="_blank" title="sc-pert" style="margin: 10px;">
<img src="figures/index/sc-pert.png" alt="sc-pert Logo" style="width:150px; height:auto; border:0;">
</a></p>
<p><a href="[https://pertpy.readthedocs.io/en/stable/](https://pertpy.readthedocs.io/en/stable/)" target="_blank" title="pertpy" style="margin: 10px;">
<img src="figures/index/pertpy.svg" alt="pertpy Logo" style="width:150px; height:auto; border:0;">
</a></p>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
